File: app/src/main/java/pl/mareklangiewicz/myblockssample/Fragment2.java
package pl.mareklangiewicz.myblockssample;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import pl.mareklangiewicz.myfragments.MyFragment;

public class Fragment2 extends MyFragment {

    @Nullable @Override public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        inflateMenu(R.menu.local_menu2);
        return inflater.inflate(R.layout.fragment2, container, false);
    }
}

File: app/src/main/java/pl/mareklangiewicz/myblockssample/Fragment2.kt
package pl.mareklangiewicz.myblockssample

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import pl.mareklangiewicz.myfragments.MyFragment

class Fragment2 : MyFragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        manager?.lnav?.menuId = R.menu.local_menu2
        return inflater.inflate(R.layout.fragment2, container, false)
    }
}

